crates/red_knot_python_semantic/resources/mdtest/literal/string.md

```diff
@@ -4,15 +4,15 @@
 
 ```py
 reveal_type("Hello")  # revealed: Literal["Hello"]
-reveal_type('world')  # revealed: Literal["world"]
-reveal_type("Guten " + 'Tag')  # revealed: Literal["Guten Tag"]
-reveal_type('bon ' + "jour")  # revealed: Literal["bon jour"]
+reveal_type("world")  # revealed: Literal["world"]
+reveal_type("Guten " + "Tag")  # revealed: Literal["Guten Tag"]
+reveal_type("bon " + "jour")  # revealed: Literal["bon jour"]
 ```
 
 ## Nested Quotes
 
 ```py
 reveal_type('I say "hello" to you')  # revealed: Literal["I say \"hello\" to you"]
-reveal_type("You say \"hey\" back")  # revealed: Literal["You say \"hey\" back"]
+reveal_type('You say "hey" back')  # revealed: Literal["You say \"hey\" back"]
```

Comment:
here we're not testing what we wanted to test anymore

Reply:
I fixed this with a judicious use of `# fmt: skip`. Don't tell @JelleZijlstra.

Reply:
Then don't mention me :) If you're ever going to use `fmt: skip` this seems like the best reason.

Reply:
> Then don't mention me :)

hehe, sorry, that was an attempt at a joke 😄